### ISSUE TITLE
Placeholder styled are not applied after bumping from 3.4.5 to 3.4.7

### ISSUE THREAD

**Issue by BJJLangedijk** [Roman86/tailwindcss-scoped-preflight#60]
**Describe the bug**

It seems like placeholder styles are no longer applied/grouped correctly. I am now seemingly missing classes when compiling the Tailwind.

This is the class I've used to style to placeholder property:
`placeholder:text-red aria-disabled:placeholder:text-blue`

I believe the issue can be found around here:

https://github.com/Roman86/tailwindcss-scoped-preflight/commit/29b188be117173636714407c30c5d102ceb4c464?diff=split&w=1#diff-dffa5e4444cd27132e014e5677f644bc69bf6a5978f9aa58d11b3dc94134a54aR29

Since placeholder is also defined as `::placeholder` it is now treated as `::before, ::after`, etc.

Currently I think it just defaults to the `color: #9ca3af;` which is in the Tailwind reset.

**Expected behaviour**
I expect the placeholder to be red by default and blue when `aria-disabled` is active.

**Screenshots**

Note: the colors are different here since they are specific colors for our implementation. 

3.4.7.
![image](https://github.com/user-attachments/assets/24107cf9-b3ee-4055-bb7d-09b27772800d)

3.4.5
![image](https://github.com/user-attachments/assets/4e2e1b41-9abf-4928-96ef-6d19ad787651)

**Comment by Roman86** [comment]
hi @BJJLangedijk ! Thanks for bringing this up. I will fix it in minutes. Maybe you can think of some more pseudo-elements to be handled in the same way?

**Comment by Roman86** [comment]
maybe I have to skip only `::before` and `::after` speudo-elements. Not all starting with `::`

**Comment by BJJLangedijk** [comment]
Yea I would only apply it to `::before` and `::after`.

Doing a quick search I also found this to be replaced

```
::-webkit-inner-spin-button:where(.foo-tw,.foo-tw *),
::-webkit-outer-spin-button:where(.foo-tw,.foo-tw *) {
```

```
.foo-tw ::-webkit-inner-spin-button,
.foo-tw ::-webkit-outer-spin-button {
```

But I'm not sure which version is the working one as we are not using it :)

**Comment by Roman86** [comment]
I will test with those too and find out. Thanks for your help

**Comment by Roman86** [comment]
I think I found a better way to handle all of them - there are a lot. Testing, seems good so far

**Comment by BJJLangedijk** [comment]
`3.4.9` Seems to work great again 😄 

Thanks for your amazingly quick response

I don't actually see the commits/version bump in GitHub but I'm sure you'll add them soon as well

**Comment by r4m-roman** [comment]
@BJJLangedijk yes, had to push it sooner so you could see the changes too. Done

**Comment by Roman86** [comment]
@BJJLangedijk yes, had to push it sooner so you could see the changes too. Done
